cmd/kube-score/main.go

```diff
@@ -242,7 +242,10 @@ func listChecks(binName string, args []string) {
 	fs := flag.NewFlagSet(binName, flag.ExitOnError)
 	printHelp := fs.Bool("help", false, "Print help")
 	setDefault(fs, binName, "list", false)
-	fs.Parse(args)
+	err := fs.Parse(args)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I used this pattern in all places where I fixed the warnings from `errcheck`. Most likely there are fancier ways to do this; however, I saw some use of this in other places of the code. I suppose it's equivalent in many ways to `try/except: raise` in Python (and other OOP languages that have a `try/catch` statement).

Reply:
Nice fixes!

Generally, I think that it's a good practice to only allow panics in the `main()` function. Other functions should instead return an error if they fail.

So for the `listChecks` function, modify it to return an error, and change this to a `if err != nil { ... }`.